pixi.toml

```diff
@@ -14,6 +14,7 @@ install = { depends-on = "pre-commit_install"}
 pre-commit_install = "pre-commit install"
 
 [dependencies]
+cartopy = "*"
```

Comment:
is there an optional field for pixi, similar to the ability to do `pip install sweets[plotting]`?

Reply:
I have moved it to the plotting feature.